liquibase-core/src/main/java/liquibase/database/DatabaseConnection.java

```diff
@@ -39,4 +39,6 @@
     boolean isClosed() throws DatabaseException;
 
     void attached(Database database);
+
+    String getSchema();
```

Comment:
Since we are only using this right now on a JdbcConnection instance, I think this would be better to put on the JdbcDatabaseConnection object instead of on the base interface which 3rd parties may have extended from and will keep their code from compiling.

Reply:
Ok

Reply:
Refactored.